front/app/containers/Admin/pages/EditPageForm.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
```

Comment:
Look at the deleted Edit/index.tsx if you better want to understand the changes.